include/AlgTraits.h

```diff
@@ -22,6 +25,8 @@ struct AlgTraitsHex8 {
   static constexpr int numScvIp_ = 8;
   static constexpr int numGp_ = 8; // for FEM
   static constexpr stk::topology::topology_t topo_ = stk::topology::HEX_8;
+  using            masterElementScs_ = HexSCS;
+  using            masterElementScv_ = HexSCV;
```

Comment:
For types we have been using a a captialized name in Nalu-Wind, should we adopt that approach here also? And should we drop the trailing underscore as this is a typedef and not a member of the struct? Would like to hear what would be the best practice for this.